crates/bevy_picking/src/backend.rs

```diff
@@ -41,11 +41,19 @@ use bevy_reflect::Reflect;
 pub mod prelude {
     pub use super::{ray::RayMap, HitData, PointerHits};
     pub use crate::{
+        backend::Pickable,
         pointer::{PointerId, PointerLocation},
         PickSet, PickingBehavior,
     };
 }
 
+/// Component used to mark entities as pickable by backends.
+///
+/// You should use this component if you are using a backend with opt-in behavior.
+#[derive(Component, Reflect, Debug)]
+#[reflect(Component, Debug)]
+pub struct Pickable;
```

Comment:
This type existed in mod_picking before upstream. It then turned into `PickingBehavior`, an optional component. We should discuss before adding this back.